Description: install dtterm terminfo into DESTDIR
 The dtterm install hook called tic without an output directory, which caused
 tic to try writing to the builder home directory during package builds.
 Create the staged terminfo directory and direct tic output there instead.
Author: Reaper <JohnGrimmReaper@disroot.org>
Forwarded: not-yet
Last-Update: 2026-03-22

--- a/programs/dtterm/Makefile.am
+++ b/programs/dtterm/Makefile.am
@@ -52,4 +52,5 @@ if !HAS_UTEMPTER_LIBRARY
 	chown root $(DESTDIR)$(bindir)/dtterm
 	chmod 4755 $(DESTDIR)$(bindir)/dtterm
 endif
-	@TIC@ dtterm.ti
+	$(MKDIR_P) $(DESTDIR)$(datadir)/terminfo
+	@TIC@ -o $(DESTDIR)$(datadir)/terminfo dtterm.ti
